docs/data/date-pickers/getting-started/getting-started.md

```diff
@@ -21,31 +21,38 @@ packageName: '@mui/x-date-pickers'
 
 ### Setup
 
-You need to provide a date-library that is used by the pickers by setting the `dateAdapter` to an adapter of your choosing.
+#### Package installation
 
-We currently support 4 different date-libraries:
+You need to install 3 different types of package to make the pickers work:
 
-- [date-fns](https://date-fns.org/)
-- [Day.js](https://day.js.org/)
-- [Luxon](https://moment.github.io/luxon/#/)
-- [Moment.js](https://momentjs.com/)
+1. **The component** (`@mui/x-date-pickers` or `@mui/x-date-pickers-pro`) which manages the rendering.
+2. **The date-library** ([`moment`](https://momentjs.com/), [`dayjs`](https://day.js.org/), ...) which manages the date manipulation.
+3. **The adapter** ([`@date-io`](https://github.com/dmtrKovalenko/date-io#projects)) expose your favorite **date-library** under a unified api used by **component**.
 
-If you need to use `js-joda`, `date-fns-jalali`, `jalaali`, or `hijri` library, you should be able to find the corresponding date-library from [`@date-io`](https://github.com/dmtrKovalenko/date-io#projects).
+First you have to install the date-library you want to use to manage dates, and the component package:
 
-First you have to install the adapter package for the date-library you want to use:
+{{"demo": "Instructions.js", "bg": "inline", "hideToolbar": true, "disableAd": true}}
 
-```sh
-// date-fns
-npm install @date-io/date-fns
-// or for Day.js
-npm install @date-io/dayjs
-// or for Luxon
-npm install @date-io/luxon
-// or for Moment.js
-npm install @date-io/moment
+We currently support 4 different date-libraries:
+
+- [date-fns](https://date-fns.org/) adapted by `@date-io/date-fns`.
+- [Day.js](https://day.js.org/) adapted by `@date-io/dayjs`.
+- [Luxon](https://moment.github.io/luxon/#/) adapted by `@date-io/luxon`.
+- [Moment.js](https://momentjs.com/) adapted by `@date-io/moment`.
+
+If you need to use `js-joda`, `date-fns-jalali`, `moment-jalaali`, or `moment-hijri` library, you should be able to find the corresponding date-library from [`@date-io`](https://github.com/dmtrKovalenko/date-io#projects).
+In such a case, you will have to install both the _date-library_ and the corresponding `@date-io` adapter.
+
+```jsx
+// To use moment-jalaali
+yarn add moment-jalaali
+yarn add @date-io/jalaali
```

Comment:
Maybe add a section to fully cover https://github.com/mui/mui-x/issues/5290 saying that if date-io does not contain any adapter for your date library, you can create one and the date-io team will help you ?

Reply:
I added the section